packages/icon-button-toggle/src/icon-button-toggle-base.ts

```diff
@@ -55,6 +55,13 @@ export class IconButtonToggleBase extends BaseElement {
   }
 
   protected handleClick() {
+    const onIconSlotted = this.querySelector('[slot="onIcon"]');
+    const offIconSlotted = this.querySelector('[slot="offIcon"]');
+
+    if (!(this.onIcon || onIconSlotted) || !(this.offIcon || offIconSlotted)) {
```

Comment:
Why is this check needed? Foundation blows up or something? Maybe add a comment?

Reply:
There were tests that were written that checked for this behavior. It seemed like it was missing. Thanks for pointing it out because I wanted to know if you thought if we should remove those tests or add the functionality back in?

Reply:
I think I'm going to revert this behavior